### ISSUE TITLE
Error Propagation

### ISSUE THREAD

**Issue by vegtelenseg** [kirill578/react-native-webview-bridge-seamless#1]
Hello,

Thanks for the work on this bridge.

I have realised that the `async` used [here](https://github.com/kirill578/react-native-webview-bridge-seamless/blob/master/src/withWebViewBridge.tsx#L36) may cause unexpected surprises and that should be removed.

Consider this for an example: 

```
let p = new Promise(resolve => {
  ""(); // TypeError
  resolve();
});

(async () => {
  await p;
})().catch(e => console.log("Caught: " + e)); // Catches the error
```

and this, similar to the code I have referenced in the link above

```
let p = new Promise(async resolve => {
  ""(); // TypeError
  resolve();
});

(async () => {
  await p;
})().catch(e => console.log("Caught: " + e)); // Doesn't catch it!
```

I have opened a [PR](https://github.com/kirill578/react-native-webview-bridge-seamless/pull/2) to resolve this.

Thanks

**Comment by kirill578** [comment]
Nice find, published as `0.0.9`. Have you experienced any issues because of this `async`?

**Comment by vegtelenseg** [comment]
Hello,

I have not experienced any issues with it, but I have had this kind of problem with promises whenever I use them like that
